cedar-policy/src/api.rs

```diff
@@ -2110,13 +2247,37 @@ pub struct Context(ast::Context);
 
 impl Context {
     /// Create an empty `Context`
+    /// ```
+    /// use cedar_policy::Context;
+    /// let c = Context::empty();
+    /// // let request: Request = Request::new(Some(principal), Some(action), Some(resource), c);
+    /// ```
     pub fn empty() -> Self {
         Self(ast::Context::empty())
     }
 
     /// Create a `Context` from a map of key to "restricted expression",
     /// or a Vec of `(key, restricted expression)` pairs, or any other iterator
     /// of `(key, restricted expression)` pairs.
+    /// ```
+    /// use cedar_policy::{Context,RestrictedExpression};
+    /// use std::collections::HashMap;
+    /// use std::str::FromStr;
+    /// let t = r#"{
```

Comment:
same comment here about our best-practice to use `json!` macro instead of a string

Reply:
done